src/main/haskell/kore/src/Kore/Attribute/Exit.hs

```diff
@@ -0,0 +1,72 @@
+{-|
+Module      : Kore.Attribute.Exit
+Description : Exit symbol attribute
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : thomas.tuegel@runtimeverification.com
+
+-}
+module Kore.Attribute.Exit
```

Comment:
Is this used anywhere besides its own test?

Reply:
No, my first implementation depended on it, but it turns out you don't need to know the exit cell.  Should I remove it?

Reply:
Unused code tends to break with time, so I would remove it unless someone else has some reason to keep it.

Reply:
Okay, I'll get rid of it.